db_backup.sh

```diff
@@ -9,15 +12,16 @@ cat <<-END
 How to make a backup of n3c schema
 ----------------------------------
 
-# sed: Changes all instances of 'n3c' in the pg_dump file to $fname, but only where 'n3c' appears as a schema name. It does this by looking at any line that isn't a line of data (all lines of data start with a number).
-
+Step 1: Backup the DB
 pg_dump -d \$psql_conn -n n3c | sed '/^[0-9][0-9]*\t/! s/[[:<:]]n3c[[:>:]]/$fname/' > $fname.dmp
 
-Then, immediately upload the backup schema to the database:
-
+Step 2: Upload
+Immediately upload the backup schema to the database.
 psql -d \$psql_conn < $fname.dmp
 
-Make sure that the dump file is around 7.9G and that it restored appropriately.
+Step 3: Quality control checks
```

Comment:
@SigfriedUpdated `db_backup.sh`
- New "quality control checks" section.
- Changed expected db size from 7.9G ->7.6G, as this is the current status. I'm not sure if it makes sense to keep this figure here though, as the DB size is gonna keep changing over time.
- Re-formatted the rest of the script to make it easier to follow and read.